java/apirag-openai/src/main/java/com/datasqrl/ai/models/groq/GroqChatProvider.java

```diff
@@ -3,10 +3,9 @@
 import com.datasqrl.ai.backend.ChatSessionComponents;
```

Comment:
Why is the GroqChatProvider in the same module as openai? There is no duplicate code if we put them in two different modules, and they are two different providers.